askbot/models/post.py

```diff
@@ -173,6 +173,8 @@ def get_answers(self, user=None):
         optionally filtered to exclude posts of groups
         to which user does not belong"""
         answers = self.filter(post_type='answer')
+        if user == 'n/a':
```

Comment:
Would value `None` work here? Also, perhaps it would be better to modify method `PostManager.get_for_user` ?
I see that the get_for_user is filtering by group...

At last the value 'n/a' looks hackish.